docker/ol-home-start.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# quick method to start all ol services from one script
+# inside an container, bypass all upstart/services
+
+CONFIG=conf/openlibrary.yml
+
+python --version
+
+if [[ "$INFOGAMI" = "local" ]]; then
+  # If we're using the local copy of infogami, don't pull anything
+  # and just use what's there
+  echo "Using local infogami"
+else
+  if [[ "$PYENV_VERSION" = 3* || -n $INFOGAMI ]]; then
+    pushd vendor/infogami
+    # Use Python 3 compatible infogami
+    git pull origin "${INFOGAMI:-master}"
+    popd
+  else
+    # Use production infogami
```

Comment:
Is 'production Infogami' accurate? Isn't it selecting a commit or tag rather than a running instance here?

Reply:
Currently, this is the Infogami that is used on Production.  (vs. Dev, Staging, etc.)

Reply:
```suggestion
    # Use version of infogami on production
```